corehq/motech/repeaters/models.py

```diff
@@ -249,6 +249,7 @@ def get_queryset(self):
 class SQLRepeater(SyncSQLToCouchMixin, RepeaterSuperProxy):
     domain = models.CharField(max_length=126, db_index=True)
     repeater_id = models.CharField(max_length=36, unique=True)
+    format = models.CharField(max_length=64, null=True, blank=True)
```

Comment:
Seems fine to add this column if it's used by all repeaters. The value of `blank` probably does not matter; I think it can be removed since it only applies to form validation.

Since this will be applied as a migration on existing data I think we must start out with `null=True`. If we want to make it NOT NULL we could add a migration to update all existing NULL rows and then add a NOT NULL constraint to the column.

For the repeaters that do not have it set (meaning the value is NULL in Postgres), should it default to TRUE or FALSE or is it different depending on the repeater?

Reply:
> For the repeaters that do not have it set (meaning the value is NULL in Postgres), should it default to TRUE or FALSE or is it different depending on the repeater?

I did not fully understand the question. If you meant to ask the default what the default value of `format` would be then the answer is that it would depend on the type of repeater and would not be a boolean.

Reply:
Huh, why was I thinking this was a boolean field? Yes, my question did not make any sense. Ignore it.